pico_limbo/src/server_state.rs

```diff
@@ -277,6 +288,20 @@ impl ServerStateBuilder {
         self
     }
 
+    pub fn tablist<S1, S2>(&mut self, header: S1, footer: S2) -> &mut Self
+    where
+        S1: AsRef<str>,
+        S2: AsRef<str>,
+    {
+        let header = parse_mini_message(header.as_ref());
+        let footer = parse_mini_message(footer.as_ref());
+        self.tablist = TabList {
+            header: header.ok(),
+            footer: footer.ok(),
+        };
```

Comment:
If both header an footer are empty strings, `self.tablist` should be set to none so that the packet is not sent.